docs/built_in_datasets.rst

```diff
@@ -124,3 +141,43 @@ argument:
 
         H5PYDataset     0.1
         fuel.converters 0.1
+
+
+Working with external packages
+------------------------------
+
+By default, Fuel looks for downloaders and converters in the
+``fuel.downloaders`` and ``fuel.converters`` modules, respectively, but you're
+not limited to that.
+
+Fuel can be told to look into additional modules by setting the
+``extra_downloaders`` and ``extra_converters`` configuration variables in
+``~/.fuelrc``. These variables are expected to be lists of module names.
+
+For instance, suppose you'd like to include the following modules:
+
+* ``package1.extra_downloaders``
+* ``package2.extra_downloaders``
+* ``package1.extra_converters``
+* ``package2.extra_converters``
+
+You should include the following in your ``~/.fuelrc``:
+
+.. code-block:: yaml
+
+    # ~/.fuelrc
+    extra_downloaders: ['package1.extra_downloaders', 'package2.extra_downloaders']
```

Comment:
This works with the YAML list syntax too, e.g.
```
extra_downloaders:
- package1.extra_downloaders
- package2.extra_downloaders
extra_converters:
- package1.extra_converters
- package2.extra_converters
```
Might be more readable from a docs perspective.